src/units.jl

```diff
@@ -252,6 +260,16 @@ end
 # Do not wish to define Gaussian units, as it changes
 # some formulas. Safer to force user to work exclusively in one unit system.
 
+## Energy
+@_lazy_register_unit cal 4.184 * J
```

Comment:
How important are calories to you? Since the unit is ambiguous ("small calorie" vs "large calorie") I wonder if we should instead have the user use only Joules, which is more explicit.